jupyterlab-server-proxy/package.json

```diff
@@ -41,18 +41,20 @@
     "install:extension": "jupyter labextension develop --overwrite .",
     "prepare": "jlpm run clean && jlpm run build:prod",
     "watch": "run-p watch:src watch:labextension",
-    "watch:src": "tsc -w",
+    "watch:src": "tsc -w --preserveWatchOutput",
     "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
     "@jupyterlab/application": "^2.0.0 || ^3.0.0",
     "@jupyterlab/apputils": "^2.0.0 || ^3.0.0",
-    "@jupyterlab/launcher": "^2.0.0 || ^3.0.0"
+    "@jupyterlab/launcher": "^2.0.0 || ^3.0.0",
+    "@jupyterlab/mainmenu": "^2.0.0 || ^3.0.0"
   },
   "devDependencies": {
     "@jupyterlab/builder": "^3.0.2",
-    "rimraf": "^2.6.1",
-    "typescript": "~3.7.0"
+    "rimraf": "^3.0.2",
+    "typescript": "~4.2.4",
+    "npm-run-all": "~4.1.5"
```

Comment:
makes `watch` work as advertised. the typescript/rimraf stuff is just a ride-along.